CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.2.3 (2019-05-03)
+
+- Fixed date range picker to properly represent Angular form control statuses (`dirty`, `pristine`, etc.). [#44](https://github.com/blackbaud/skyux-datetime/pull/44)
```

Comment:
"date range picker [component]" (to be consistent with other lines)?

Reply:
Done.